### ISSUE TITLE
Use workspaces name provided by hyprland

### ISSUE THREAD

**Issue by kRHYME7** [Jas-SinghFSU/HyprPanel#358]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Option to use this to have workspace Icons/name etc 
``` bash 
hyprctl workspaces -j | jq -r  '.[].name'
```
![image](https://github.com/user-attachments/assets/77ca2df6-0b53-4792-b8b3-f59d4be5a5ee)
Comparison
![image](https://github.com/user-attachments/assets/6cc5a193-0045-46b7-baa1-dbe0f0f96587)

With this we can just declare anything under hyprland configuration and basically the panel will treat it as native. (no changes over hyprpanel needed for wokspaces )

**Describe alternatives you've considered**

Using 
``` bash 
hyprctl workspaces -j | jq -r  '.[].name'
```

But this might be a bad approach for what ever the project philosophy. So I would like to request to have a toggle option to enable using  hyprctl's output. 

**Additional context**

I don't have any changes in waybar in this case. These are all done using hyprland.conf. 
My take is having options in the setting dialog is great  but using  hyprlands own identifiers is seamless. 

![image](https://github.com/user-attachments/assets/d34e1192-4cae-4433-891d-e34f67a502c2)

The hyprland configuration I had
![image](https://github.com/user-attachments/assets/7ee70a92-e6f7-425b-ab0d-1c5645b31395)

**Comment by rubiin** [comment]
we currently have two styles. Numbered and a circular indicator. Should this be a third option ?

**Comment by kRHYME7** [comment]
I hope so. For me I'm accustomed to named workspaces, so this is a beneficial feature. Also this one should be seamless to hyprland workspaces.

**Comment by rubiin** [comment]
Yeah . We can added it as a third option similar to numbered. Maybe like "Show native workspace names" toggle
